code/modules/clothing/under/syndicate.dm

```diff
@@ -50,6 +50,7 @@
 	icon_state = "bloodred_pajamas"
 	inhand_icon_state = "bl_suit"
 	armor_type = /datum/armor/clothing_under/bloodred_sleepytime
+	supports_variations_flags = CLOTHING_DIGITIGRADE_VARIATION
```

Comment:
This needs to be done modularly; as-is, this is editing a TG file directly. Bad for a downstream, will cause merge conflicts whenever TG touches this file.

Look under the modular_skyrat folder for somewhere relevant to put this. A maintainer will probably have an idea of where exactly that might be; IIRC there was a modular_overrides file buried in that mess somewhere for all the cases we added variation flags to items.

Reply:
will be a lesson to me not to do everything in one PR

Reply:
done